fixtures/eurolite/z-par.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Z-PAR",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["LiMa"],
+    "createDate": "2023-11-23",
+    "lastModifyDate": "2023-11-23"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/download/51916501-Anleitung-136973-1.200-eurolite-led-z-par-rgbw-4x10w-de_en.pdf"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/de/mpn51916501-eurolite-led-z-par-rgbw-4x10w.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=jHNNSnWji4g&t=4s"
+    ]
+  },
+  "physical": {
+    "bulb": {
+      "type": "LED RGBW"
+    },
+    "lens": {
+      "name": "Zoom",
+      "degreesMinMax": [6, 49]
+    }
+  },
+  "availableChannels": {
+    "Zoom": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Zoom",
+        "angleStart": "narrow",
+        "angleEnd": "wide"
```

Comment:
The manual says "from near to far", which I would interpret as wide-to-narrow, but I don't have a device to test.